extensions-core/core/deployment/src/main/java/org/apache/camel/quarkus/core/deployment/CamelNativeImageProcessor.java

```diff
@@ -217,31 +217,13 @@ void embedSelectResourcesInNativeExecutable(CamelConfig config, ApplicationArchi
             BuildProducer<NativeImageResourceBuildItem> resources) {
 
         final ResourcesConfig resourcesConfig = config.native_.resources;
-        if (!resourcesConfig.includePatterns.isPresent()) {
-            LOGGER.debug("Not scanning resources for native inclusion as include-patterns is not set");
-            return;
+        if (resourcesConfig.includePatterns.isPresent()) {
+            throw new IllegalStateException(
+                    "quarkus.camel.native.resources.include-patterns configuration property was removed in Camel Quarkus 2.0.0. Use quarkus.native.resources.includes instead.");
```

Comment:
If we break in camel-quarkus 2.0, then maybe it's a good candidate for the migration guide ?

Reply:
Yes, done in docs/modules/ROOT/pages/migration-guide/2.0.0.adoc